x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/calculations/moving_average.tsx

```diff
@@ -168,3 +169,67 @@ function MovingAverageParamEditor({
     </EuiFormRow>
   );
 }
+
+const MovingAveragePopup = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  return (
+    <HelpPopover
+      anchorPosition="upCenter"
+      button={
+        <HelpPopoverButton onClick={() => setIsPopoverOpen(!isPopoverOpen)}>
+          {i18n.translate('xpack.lens.indexPattern.movingAverage.helpText', {
+            defaultMessage: 'How does it work?',
```

Comment:
I suggest that we use this format for the text throughout:

How it works
How moving average works